### DISCUSSION TITLE
how to disable subtitles?

### DISCUSSION THREAD

**Discussion by tarkovshoota** [rocksdanister/lively#1843] [Q&A] [answered]
i have searched a good bit on how to disable subtitles, cant find anything, i thought it was my YouTube downloader the whole time but i opened the video and their were no subtitles, i know when you click preview you can disable audio track and subtitle track which does what i want but it doesn't actually let you save it to my knowledge. basically i just want to know how to disable all the text that shows up automatically through lively because it gets annoying on my monitors, theirs just random auto generated lyrics.

**Comment by rocksdanister** [comment, marked as answer]
Didn't know mpv loaded subs by default, these flag should disable it:
https://mpv.io/manual/stable/#options-no-sub-ass
https://mpv.io/manual/stable/#options-no-sub-auto

Create a mpv configuration file in appdata folder:
https://github.com/rocksdanister/lively/wiki/Video-Guide#mpv

```
no-sub-ass
no-sub-auto
```

**Reply by tarkovshoota** [reply]
thanks man! love the software its awesome
